### ISSUE TITLE
Feature Request: Installation of git dependencies

### ISSUE THREAD

**Issue by Yarn** [pleiszenburg/wenv#20]
It would be nice to be able to install git dependencies in wenv. I suspect the only thing needed for this to work is to have git installed inside the wine bottle.

**Comment by s-m-e** [comment]
Indeed, this would a nice-to-have feature. PR welcome, else added to my to-do list but it might take a while.

**Comment by Yarn** [comment]
i figured out how to install git in the wenv wine environment

fish shell code
```fish
# replace `~/venv/` with path to your virtual environment
set -x WINEPREFIX ~/venv/share/wenv/win32/
# can also download manually
wget 'https://github.com/git-for-windows/git/releases/download/v2.42.0.windows.1/Git-2.42.0-32-bit.exe'

# run through the installer gui
# for me it hung indefinitely at the end of the install
# but it seems to have finished the install before hanging
wine Git-2.42.0-32-bit.exe
```

a normal `wenv pip install <package>` will then work with git dependencies

**Comment by marc-hb** [comment]
I got "MinGit" to work by merely extracting the Windows .zip file from Linux into `~/.local/share/wenv/win32/drive_c/MinGit`.
Didn't even need to run an any .exe installer. Nice!

On the other hand, I had to place this hack in my Python code, less nice:
```
gitdir = r'c:\MinGit\cmd'
os.environ['PATH'] += f';{gitdir}'
```
I searched everywhere but couldn't find how to change the Windows PATH?
